packages/webapp/src/containers/Task/index.jsx

```diff
@@ -170,47 +187,60 @@ export default function TaskPage({ history }) {
-        <TasksFilterPage onGoBack={onFilterClose} />
-      </Drawer>
-      {isFilterCurrentlyActive && (
-        <div style={{ marginBottom: '32px' }}>
-          <ActiveFilterBox pageFilter={tasksFilter} pageFilterKey={'tasks'} />
-          <div style={{ marginTop: '12px' }}>
-            <Underlined style={{ color: '#AA5F04' }} onClick={resetFilter}>
-              {t('FILTER.CLEAR_ALL_FILTERS')}
-            </Underlined>
+    <>
+      <Layout>
+        <PageTitle title={t('TASK.PAGE_TITLE')} style={{ paddingBottom: '20px' }} />
+        <PureTaskDropdownFilter
+          onDateOrderChange={onDateOrderChange}
+          isAscending={tasksFilter[IS_ASCENDING]}
+          onAssigneeChange={onAssigneeChange}
+          assigneeValue={assigneeValue}
+          onFilterOpen={onFilterOpen}
+          isFilterActive={isFilterCurrentlyActive}
+        />
+        <TaskCount count={taskCardContents.length} />
+        <Drawer title={t('TASK.FILTER.TITLE')} isOpen={isFilterOpen} onClose={onFilterClose}>
+          <TasksFilterPage onGoBack={onFilterClose} />
+        </Drawer>
+        {isFilterCurrentlyActive && (
+          <div style={{ marginBottom: '32px' }}>
+            <ActiveFilterBox pageFilter={tasksFilter} pageFilterKey={'tasks'} />
+            <div style={{ marginTop: '12px' }}>
+              <Underlined style={{ color: '#AA5F04' }} onClick={resetFilter}>
+                {t('FILTER.CLEAR_ALL_FILTERS')}
+              </Underlined>
+            </div>
           </div>
-        </div>
-      )}
+        )}
+
+        {taskCardContents.length > 0 ? (
+          taskCardContents.map((task) => (
+            <TaskCard
+              key={task.task_id}
+              onClick={() => history.push(`/tasks/${task.task_id}/read_only`)}
+              style={{ marginBottom: '14px' }}
+              {...task}
+            />
+          ))
+        ) : (
+          <Semibold style={{ color: 'var(--teal700)' }}>{t('TASK.NO_TASKS_TO_DISPLAY')}</Semibold>
+        )}
+      </Layout>
 
-      {taskCardContents.length > 0 ? (
-        taskCardContents.map((task) => (
-          <TaskCard
-            key={task.task_id}
-            onClick={() => history.push(`/tasks/${task.task_id}/read_only`)}
-            style={{ marginBottom: '14px' }}
-            {...task}
-          />
-        ))
-      ) : (
-        <Semibold style={{ color: 'var(--teal700)' }}>{t('TASK.NO_TASKS_TO_DISPLAY')}</Semibold>
+      {createLocation && (
+        <LocationCreationModal
+          title={t('LOCATION_CREATION.TASK_TITLE')}
+          body={
+            isAdmin ? t('LOCATION_CREATION.TASK_BODY') : t('LOCATION_CREATION.TASK_BODY_WORKER')
+          }
+          dismissModal={dismissLocationCreationModal}
+          isAdmin={isAdmin}
+        />
       )}
-    </Layout>
+
+      <div className={styles.buttonWrapper}>
+        <FloatingActionButton type={'add'} onClick={handleAddTask} />
```

Comment:
I think you brought this up before, but do we want the button to trigger the action directly, or to open the floating menu like we're doing with Animals? It does make sense to just trigger the action if there's only one option, but we could double check with Loïc (if we haven't) and if this is the way make it consistent in Animals as well.

Reply:
Thanks for bring it up! As discussed with Loïc today, I'll make them both one-click (no menu).